File: submits/readability/readability.c
#include <ctype.h>
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int count_letters(string text);
int count_words(string textb);

int main(void)
{
    string text1 = get_string("Write text sample: ");
    int lett = count_letters(text1);
    printf("%i letters\n", lett);
    int word = count_words(text1);
    printf("%i words\n", word);
}

int count_letters(string text)
{
    int n = strlen(text);
    int letters = 0;
    for (int i = 0; i < n; i++ )
    {
        if(isupper(text[i]))
        {
            letters += 1;
        }
        if(islower(text[i]))
        {
            letters += 1;
        }
    }
    return letters;
}

int count_words(string textb)
{
    int n = strlen(textb);
    int words = 0;
    for (int i = 0; i < n; i++ )
    {
        if (strcmp(textb[i], " ") == 0)
        {
            words += 1;
        }
    }
    return words;
}

File: submits/sentimental-readability/readability.py
# TODO

from cs50 import get_string

def main():
    text1 = get_string("Write text sample: ")
    letters = count_letters(text1)


def count_letters(text):
    n = len(text)
    letters = 0
    for i in range(n):
        if text[i] in range (65, 90):
            letters += 1
        if text[i] in range (65, 90):
            letters += 1
    return letters


if __name__ == "__main__":
    main()